sbin/drive-format.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+if [[ "${1}" == "--check" ]]
+    # Use return code of "0" if it is safe to format or "1" if potentially unsafe.
+    then drive_format_check_rc=0
+    # The partition type reported by `parted` is one of three possible values: "gpt", "msdos" (MBR), or "unknown" (none).
+    if parted ${2} print 2> /dev/null | grep -q "Partition Table: unknown"
+        then echo "INFO: Partition table does not exist."
+    else
+        echo "WARNING: Partition table does exist!"
+        drive_format_check_rc=1
```

Comment:
Aren't we done at this point? `drive_format_check_rc` will be 1 no matter what, so no need for all the complexity below?

No one is intended to use this script directly, so all the logic to provide detailed information seems unnecessary.